packages/slate/test/interfaces/Range/equals/other-props-not-equal.tsx

```diff
@@ -0,0 +1,30 @@
+import { Range } from 'slate'
+
+export const input = {
```

Comment:
Maybe also add a test when the ranges has a different number of keys they should also not equal and on both sides. Just because it's working now it might not work tomorrow if someone tries to optimize that equals code or something.